app/src/components/Shared/Modal.jsx

```diff
@@ -43,9 +43,12 @@ class Modal extends React.Component {
 
   render() {
     if (!this.props.opened) {
+      document.body.style.overflow = 'overlay';
```

Comment:
State styles should be applied with a class and defined in CSS. The convention is `.-hidden` or `.-expanded` for example.
(not saying it's necessarily better, but it's the convention in this project, let's try to be consistent)